.env

```diff
@@ -1,4 +1,4 @@
-_APP_ENV=development
+_APP_ENV=production
```

Comment:
Why set to production any particular reason?

Reply:
No reason, tried the cert worker, but i guess I need a live deployment